keyboards/anavi/knob1/knob1.c

```diff
@@ -0,0 +1,54 @@
+// Copyright 2022 Leon Anavi <leon@anavi.org>
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "encoder.h"
+#include <stdio.h>
+
+void keyboard_post_init_kb(void) {
+    // Enable RGB LED
+    setPinOutput(GP11);
+    writePinHigh(GP11);
+    rgblight_enable();
+
+    // Offload to the user func
+    keyboard_post_init_user();
+}
+
+#ifdef ENCODER_ENABLE
+bool encoder_update_kb(uint8_t index, bool clockwise) {
+    if (!encoder_update_user(index, clockwise)) { return false; }
+    if (clockwise) {
+        tap_code(KC_VOLU);
+    } else {
+        tap_code(KC_VOLD);
+    }
+    return true;
+}
+#endif
+
+#ifdef OLED_ENABLE
+
+bool oled_task_user(void) {
+  // Host Keyboard Layer Status
```

Comment:
```suggestion
    if (!oled_task_user()) {
        return false;
    }

    // Host Keyboard Layer Status
```

Reply:
@drashna thank you for the code review. Sorry, I am a bit confused here. In this code snippet I have `oled_task_user` in `keyboards/anavi/knob1/knob1.c`. Should I change it from `oled_task_user` to `oled_task_kb` and then in it to do this check for `if(!oled_task_user())`?

Thanks, Leon

Reply:
in knob1.c, yeah, it should look like:
```c
bool oled_task_kb(void) {
    if (!oled_task_user()) {
        return false;
    }
    // oled code here

    return true;
}
```

This allows you to specify a default behavior for the oled code, and allows users to override the behavior, if they want. 

This is especially useful for those using the QMK Configurator to compile, since it allows them to have a functional OLED display, without having to touch C code.

Reply:
@drashna thank you for the provided details and for the code review. Following your recommendations I have modified `knob1.c`.

Thanks,
Leon